todoman/cli.py

```diff
@@ -224,49 +247,48 @@ def edit(ctx, id, todo_properties, interactive):
             setattr(todo, key, value)
 
     if interactive or (not changes and interactive is None):
-        ui = TodoEditor(todo, ctx.obj['db'].lists(), ctx.obj['formatter'])
+        ui = TodoEditor(todo, ctx.db.lists(), ctx.ui_formatter)
```

Comment:
TBH, I'm not very convinced by this. I'd much rather disallow `interactive` and `porcelain` together (since they don't make too much sense anyway).